rpcs3/Emu/Cell/lv2/sys_ppu_thread.cpp

```diff
@@ -69,8 +69,6 @@ void _sys_ppu_thread_exit(ppu_thread& ppu, u64 errorcode)
 
 	sys_ppu_thread.trace("_sys_ppu_thread_exit(errorcode=0x%llx)", errorcode);
 
-	ppu.state += cpu_flag::exit;
```

Comment:
Hmm, why remove flag set here? Maybe it should be moved to the end of function?

Reply:
ppu_thread_exit() does it.